example/src/release_tests/gesturizedPressable/bordersExample.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import TestingBase from './testingBase';
+
+export function BordersExample() {
+  return (
+    <>
+      <View style={styles.container}>
+        <View style={styles.square}>
+          <Text>Normal view</Text>
+        </View>
+        <TestingBase style={styles.square} />
+      </View>
+    </>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    gap: 25,
+    height: 130,
+    marginBottom: 40,
+    flexDirection: 'row',
+  },
+  square: {
+    width: 100,
+    height: 100,
+    borderRadius: 10,
+    borderWidth: 5,
+    borderBottomColor: '#0f0',
+    borderRightColor: '#f00',
+    borderLeftWidth: 1,
+    borderStyle: 'dashed',
```

Comment:
Hey just to address your issues.

> If you perform a subtle click, Pressable will stay in pressed in state. Here is how it looks like:

I'll fix that right away.

>On web if you click on Pressable and drag out of its boundaries, you'll trigger press out callback. On native platforms it doesn't happen:

I'll fix that as well.

>Let's change that to solid since iOS doesn't support dashed border as shown on the image below

The point of this example is to show how `Pressable` behaves with dashed/dotted border which is differently supported on android, web and IOS, all of which handle borders differently when you mix different styles.

Previously, `Pressable` had issues with being consistant with the legacy version, and so i added this example.
I fixed this error since, so I think I can remove the borders example all together, as all the styles are now aligned with the legacy pressable.

For more details on the borders issue, there are screens documenting these issues further up in this PR.